### ISSUE TITLE
Breaks on Python 3.8.4 with TypeError: can't apply this __setattr__ to XMLStreamClass object

### ISSUE THREAD

**Issue by tristan** [horazont/aioxmpp#342]
Python 3.8.4 was released yesterday https://www.python.org/downloads/release/python-384/
since upgrading, trying to import aioxmpp causes the following error

```
Python 3.8.4 (default, Jul 14 2020, 03:07:46) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import aioxmpp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/aioxmpp/__init__.py", line 101, in <module>
    from .errors import (  # NOQA
  File "/usr/local/lib/python3.8/site-packages/aioxmpp/errors.py", line 90, in <module>
    from . import xso, i18n, structs
  File "/usr/local/lib/python3.8/site-packages/aioxmpp/xso/__init__.py", line 616, in <module>
    from .model import (  # NOQA: F401
  File "/usr/local/lib/python3.8/site-packages/aioxmpp/xso/model.py", line 2186, in <module>
    class XSO(metaclass=XMLStreamClass):
  File "/usr/local/lib/python3.8/site-packages/aioxmpp/xso/model.py", line 1900, in __new__
    return super().__new__(mcls, name, bases, namespace)
  File "/usr/local/lib/python3.8/abc.py", line 86, in __new__
    _abc_init(cls)
  File "/usr/local/lib/python3.8/site-packages/aioxmpp/xso/model.py", line 1948, in __setattr__
    super().__setattr__(name, value)
TypeError: can't apply this __setattr__ to XMLStreamClass object
```

**Comment by horazont** [comment]
This looks so fun.

In all seriousness, thanks for reporting. I’m not sure when I’ll get around to get my hands on a 3.8.4 to test and debug this.

**Comment by horazont** [comment]
This is a regression in Python 3.8.4: https://bugs.python.org/issue41295

**Comment by horazont** [comment]
https://github.com/python/cpython/pull/21473

**Comment by horazont** [comment]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=965069

**Comment by horazont** [comment]
PR superseded by https://github.com/python/cpython/pull/21528 (merged). [Someone](https://bugs.python.org/issue41295#msg373924) did run the aioxmpp test suite, too, so I think this’ll be fixed upstream.

I hear that Python plans a hotfix release soon, so that should fix it for good.

**Comment by horazont** [comment]
Python [has released 3.8.5](https://www.python.org/downloads/) which is [already underway in Debian unstable](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=965069#15).

I haven’t verified the fix yet, if anyone can, that’d be great.

**Comment by horazont** [comment]
Verified with 3.8.5 from debian that things work now. Thanks for the report!
